internal/services/access_flow.go

```diff
@@ -477,7 +513,7 @@ func convertGranteeFilterGroupApiToTerraformModel(ctx context.Context, granteeFi
 
 	var filtersState []models.AttributeFilter
 	for _, apiFilter := range granteeFilterGroup.GetAttributeFilters() {
-		dataFilter, diagnostics := convertAttributeFiltersTerraformModelToApi(ctx, apiFilter)
+		dataFilter, diagnostics := convertAttributeFiltersApiToTerraformModel(ctx, apiFilter)
```

Comment:
why is it the reverse function here now?

Reply:
the name was incorrect